README.md

```diff
@@ -36,20 +36,20 @@ Python client library to quickly get started with the various [Watson APIs][wdc]
 * You need an [IBM Cloud][ibm-cloud-onboarding] account.
 
 ## Installation
-Note: We are moving to `ibm-watson`. All versions prior to v3.0.0 can still be found in `watson-developer-cloud`
-
 To install, use `pip` or `easy_install`:
 
 ```bash
-pip install --upgrade watson-developer-cloud
+pip install --upgrade ibm-watson
```

Comment:
🤦‍♂

Reply:
@germanattanasio this is for older versions!